README.md

```diff
@@ -2,22 +2,19 @@ PHP-MBTiles-Server
 ==================
 
 ### About
+This is a fork of bmcbride's [PHP-MBTiles-Server](https://github.com/bmcbride/PHP-MBTiles-Server) work. With a few security updates. 
```

Comment:
I do agree, was not aware that it would update in this repo. It was a change to my fork of the project. How can I revert this commit to this project, but keep it on my fork?